tests/functional/conftest.py

```diff
@@ -14,6 +18,24 @@ def department():
 
 
 @pytest.fixture
+def mock_oidc_issuer():
+    mock_server_port = get_free_port()
+    start_mock_server(mock_server_port)
+    return 'http://localhost:{port}/broker'.format(port=mock_server_port)
+
+
+@pytest.yield_fixture   # noqa: F811
+def provider(responses, mock_oidc_issuer):
+    global config
```

Comment:
This `global` is not necessary - change the `config` to a fixture and pass it in as a parameter